Database/Corrections/Automatic/sodBaseItems.lua

```diff
@@ -2824,10 +2824,6 @@ function SeasonOfDiscovery:LoadBaseItems()
         [210187] = {
             [itemKeys.name] = "Venture Co. Work Order",
             [itemKeys.npcDrops] = {3988,4070,3992,3993},
-            [itemKeys.objectDrops] = nil,
-            [itemKeys.itemDrops] = nil,
-            [itemKeys.vendors] = nil,
-            [itemKeys.startQuest] = nil,
```

Comment:
These should also stay.

Reply:
Fixed in: https://github.com/Questie/Questie/pull/5339/commits/820f3e91d4908ff1dd233f342c8907524e7e1fd3